code/modules/client/preference_setup/loadout/items/suit.dm

```diff
@@ -119,11 +119,29 @@
 	cmo_labcoats["chief medical officer labcoat, alt 2"] = /obj/item/clothing/suit/storage/toggle/labcoat/cmoalt2
 	gear_tweaks += new /datum/gear_tweak/path(cmo_labcoats)
 
-/datum/gear/suit/surgeryapron
-	display_name = "surgical apron"
-	path = /obj/item/clothing/accessory/apron/surgery
```

Comment:
Why replace the apron? Why not keep both?

I see no point in replacing/removing the apron. More options is good.

Reply:
On further thought this change should be a separate PR